docs/cugraph/source/graph_support/wholegraph_support.md

```diff
@@ -0,0 +1,4 @@
+# WholeGraph
+
+[RAPIDS](https://rapids.ai) [WholeGraph](https://github.com/rapidsai/wholegraph) is designed to help train large-scale Graph Neural Networks(GNN).
+Please see [WholeGraph Introduction](https://github.com/rapidsai/wholegraph/blob/main/docs/WholeMemoryIntroduction.md) for more details
```

Comment:
This is the page for WholeMemory, but should it instead be the main README for WholeGraph, which includes an introduction and a link to the WholeMemory page?
```suggestion
Please see [WholeGraph Introduction](https://github.com/rapidsai/wholegraph/blob/main/README.md) for more details
```